### ISSUE TITLE
[BUG] Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by SissiHermanni** [ZDoom/Raze#1054]
### Raze version

1.9.1

### Which game are you running with Raze?

None

### What Operating System are you using?

Linux x86_64

### Please describe your specific OS version

Linux Mint 20.3

### Relevant hardware info

GTX 1070

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Raze portable linux crashes with following error message in terminal: Segmentation fault (core dumped)
The last version which doesn't have this problem is 1.7.1. 

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Download Raze 1.9.1-linux-portable
2. Extract tar and try to run raze via terminal ./raze -iwad blood  || or duke.gpr, doesn't matter which game.
3. Get error message about segmentation fault.

### Provide a Log

Raze doesn't give error log for this problem.

**Comment by madame-rachelle** [comment]
Does it give you a crash log? - While it won't help for finding out what's wrong inside the program, it at least can tell us if it is inside the program or if it's one of the libraries.

**Comment by madame-rachelle** [comment]
No, you'd need gdb installed and then it would just put it out on the screen (I think). There won't be much useful info there other than telling us whether it crashes inside of Raze or inside of one of the loaded modules.

**Comment by madame-rachelle** [comment]
Don't worry about it then. Until we get more information from an actual debug build, there isn't much we can do. The Linux binaries are always stripped prior to shipping because they tend to get rather big if shipped with symbols.

**Comment by michaelneverwins** [comment]
With Raze 1.9.1 on Linux Mint 21.2, I get a segmentation fault when trying to run _Blood_ or _Cryptic Passage_, but this doesn't happen with _Duke Nukem 3D_, _Shadow Warrior_, and _Exhumed_, and doesn't happen with Raze 1.8.1, so maybe it isn't the same problem. But, in case it is the same problem, I do have a crash log to upload.

Interestingly, I don't get the crash log if I just run `raze` and let the shell find the program. For context, I unpacked `raze-v1.9.1-linux-portable.tar.xz` into `~/.local/lib/raze/raze-v1.9.1-linux-portable` and then made a link to `~/.local/lib/raze/raze-v1.9.1-linux-portable/raze` in `~/.local/bin` (which is in `$PATH`), and running `raze` generally works (and I can see that it's running version 1.9.1). However, if I run `raze` this way and select _Blood_ with this version, I get
```
*** Fatal Error ***
!!! Failed to exec debug process
Segmentation fault (core dumped)
```
in the terminal, whereas if I give the whole path (i.e. if I explicitly run either `~/.local/bin/raze` or `~/.local/lib/raze/raze-v1.9.1-linux-portable/raze`), or if I `cd` to `~/.local/lib/raze/raze-v1.9.1-linux-portable` and run `raze` there, I get
```
*** Fatal Error ***
Address not mapped to object (signal 11)
Address: 0x8
```
in the terminal and a crash log. I don't even know if the former behavior is a sign of a bug or just that I set things up badly, but I thought it was interesting.

Anyway, here's the crash log, with my username and hostname redacted (replaced with `USER` and `HOSTNAME` respectively): [`raze-crash.log`](https://github.com/ZDoom/Raze/files/13813258/raze-crash.log)

Whether this will do any good, given the previous comment and the fact that I'm not using a debug build, I don't know. But it's something.

Note that my hardware isn't the same as @SissiHermanni's. The machine I'm using right now has an AMD Radeon HD 7950 — which already seems to have trouble doing Vulkan with the portable Linux releases of Raze 1.8.1 and newer (#1017), so I'm setting my links back to Raze 1.7.1 for now.
